core/store/orm/orm_test.go

```diff
@@ -1063,7 +1067,7 @@ func TestBulkDeleteRuns(t *testing.T) {
 	var resultCount int
 	err = db.Model(&models.RunResult{}).Count(&resultCount).Error
 	assert.NoError(t, err)
-	assert.Equal(t, 6, resultCount)
+	assert.Equal(t, 3, resultCount)
```

Comment:
Why was this 6 before?

Reply:
gorm only creates records that have interesting data, they always had interesting data before because of cached values and such, but now some don't, so the initial number of records has been reduced.